public/app/percona/environment-overview/EnvironmentOverview.tsx

```diff
@@ -0,0 +1,26 @@
+import { useStyles2 } from '@grafana/ui';
+import Page from 'app/core/components/Page/Page';
+import React from 'react';
+import { PlatformConnectedLoader } from '../shared/components/Elements/PlatformConnectedLoader';
+import { usePerconaNavModel } from '../shared/components/hooks/perconaNavModel';
+import Contact from './components/ContactWidget/Contact';
+import { getStyles } from './EnvironmentOverview.styles';
+
+export const EnvironmentOverview = () => {
+  const styles = useStyles2(getStyles);
+  const navModel = usePerconaNavModel('environment-overview');
+
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents dataTestId="page-wrapper-environment-overview">
+        <PlatformConnectedLoader>
+          <div className={styles.widgetsWrapper}>
+            <Contact />
```

Comment:
It seems we only have one `Contact`. Why do we need a wrapper with grid / flex here? Is the wrapper even needed?

Reply:
Yes it is needed. Not for now, but goal is to have there multiple widgets in the future.